lib/disk/modules/QcowDisk.rb

```diff
@@ -571,6 +620,30 @@ def cluster_offset_mask
     @cluster_offset_mask ||= (1 << csize_shift) - 1
   end
 
+  def incompatible_features
+    version < 3 ? 0 : @header['incompatible_features']
+  end
+
+  def compatible_features
+    version < 3 ? 0 : @header['compatible_features']
+  end
+
+  def autoclear_features
+    version < 3 ? 0 : @header['autoclear_features']
+  end
+
+  def dirty
+    (incompatible_features & INCOMPATIBLE_FEATURES_MASK[:dirty]) == INCOMPATIBLE_FEATURES_MASK[:dirty]
```

Comment:
I think there's a `bitSet` or `bitSet?` helper method that does this... `bitSet?(incompatible_features, INCOMPATIBLE_FEATURES_MASK[:dirty]`